x-pack/solutions/security/packages/kbn-scout-security/src/playwright/fixtures/single_thread_fixtures.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { test as base } from '@kbn/scout';
+import { BrowserAuthFixture } from '@kbn/scout/src/playwright/fixtures/test/browser_auth';
+import { extendPageObjects } from './test/page_objects';
+import { SecurityBrowserAuthFixture, SecurityTestFixtures, SecurityWorkerFixtures } from './types';
+import { extendBrowserAuth } from './test/authentication';
+import { createDetectionRuleFixture } from './worker/apis/detection_rule';
+
+export const test = base.extend<SecurityTestFixtures, SecurityWorkerFixtures>({
+  browserAuth: async (
+    { browserAuth }: { browserAuth: BrowserAuthFixture },
+    use: (auth: SecurityBrowserAuthFixture) => Promise<void>
+  ) => {
+    const extendedAuth = await extendBrowserAuth(browserAuth);
+    await use(extendedAuth);
+  },
+  pageObjects: async (
+    {
+      pageObjects,
+      page,
+    }: { pageObjects: SecurityTestFixtures['pageObjects']; page: SecurityTestFixtures['page'] },
+    use: (pageObjects: SecurityTestFixtures['pageObjects']) => Promise<void>
+  ) => {
+    const extendedPageObjects = extendPageObjects(pageObjects, page);
+    await use(extendedPageObjects);
+  },
+  detectionRuleApi: [
+    async ({ kbnClient }, use) => {
+      const detectionRuleHelper = await createDetectionRuleFixture({ kbnClient });
+      await use(detectionRuleHelper);
+    },
+    { scope: 'worker' },
+  ],
```

Comment:
I would move it to its own `api` group, please check this example (you probably wan't to extend it with your fixtures)

https://github.com/elastic/kibana/tree/main/src/platform/packages/shared/kbn-scout/src/playwright/fixtures/worker/apis